### DISCUSSION TITLE
Is there a way to `forget` all targets that have been deleted in the destination directory?

### DISCUSSION THREAD

**Discussion by dylandronnier** [twpayne/chezmoi#2994] [Q&A]
Hello everyone,
I was wondering if there is a way to remove all targets that have been deleted in the destination directory from the source state (similar to how `re-add` add all targets that have been modified in the destination directory to the source state) ?
Thanks a lot for your help!

**Comment by Lockszmith-GH** [comment]
Does [this](https://github.com/twpayne/chezmoi/discussions/2810) help?
